### PR TITLE
Add support for vcf_bgzip input to gemini

### PR DESCRIPTION
**Pull request by wm75**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by mvdbeek** [comment]
Could you also take a look at the failing gemini tests (annotate and burden)  in https://mvdbeek.github.io/iuc-tool-test-results/#functional.test_toolbox.TestForTool_gemini_annotate.test_tool_000001 ?

File: tools/gemini/gemini_annotate.xml [hunk 1]
```diff
@@ -19,9 +19,19 @@
 <![CDATA[
 
     ## For GEMINI to work correctly, tabixed file must have form [name].[bed|vcf].gz
-    #set $tabixed_file = "tabixed.%s.gz" % $annotate_source.ext
-    bgzip -c '$annotate_source' > '$tabixed_file' &&
-    tabix -p '$annotate_source.ext' '$tabixed_file' &&
+    #if $annotate_source.ext == 'vcf_bgzip':
+        ## File is bgzipped already -> just tabix it
+        #set $tabixed_file = 'tabixed.vcf.gz'
```
Comment (mvdbeek) [inline tools/gemini/gemini_annotate.xml]: vcf_bgzip should already be indexed and available as `$tabixed_file.tabix_index`

Comment (wm75) [reply tools/gemini/gemini_annotate.xml]: Ah, of course! Let me fix this.

**Comment by wm75** [comment]
> Could you also take a look at the failing gemini tests (annotate and burden) in https://mvdbeek.github.io/iuc-tool-test-results/#functional.test_toolbox.TestForTool_gemini_annotate.test_tool_000001 ?

Good point! Had missed those, but will fix here (though currently I have no idea why you're getting these errors)

**Comment by mvdbeek** [comment]
The `-e +` assertion is nonsense, but I couldn't figure out what's wrong with burden. This works if you bump to 0.30.2, but then gemini load fails.

**Comment by wm75** [comment]
Yeah, a version upgrade is on my TO DO list, but likely requires quite some work.

**Comment by wm75** [comment]
@mvdbeek I have a speculative idea about gemini_burden, but to verify this: which pandas version is in the container?

**Comment by wm75** [comment]
Ah, the build date of the container is revealing enough I think. So here's my hypothesis:

The container is from June 2018 and that means it should still have a pandas version affected by this bug: https://github.com/pandas-dev/pandas/issues/21561
The relevant source code for gemini burden is:
https://github.com/arq5x/gemini/blob/cc3563fc50275d12f5af30550bf693853efbb41c/gemini/tool_burden_tests.py#L268

When I test locally after downgrading the pandas in my gemini env to pandas-0.23.1 I'm getting empty output indeed.

Question is: how do I fix this/get the container updated? Is requiring pandas at a suitable version going to help?

**Comment by mvdbeek** [comment]
> Question is: how do I fix this/get the container updated?

We could "cheat" and list an explicit pandas version for gemini burden. Else you'd have to trigger a new conda build for gemini 0.20.1.

**Comment by mvdbeek** [comment]
So 
```
diff --git a/tools/gemini/gemini_burden.xml b/tools/gemini/gemini_burden.xml
index 4300a7655..dcdfd7102 100755
--- a/tools/gemini/gemini_burden.xml
+++ b/tools/gemini/gemini_burden.xml
@@ -5,7 +5,9 @@
         <token name="@BINARY@">burden</token>
         <token name="@GEMINI_BUG_WARNING@">Caveat: due to a bug in this version of GEMINI, filtering on alternate allele frequency will get disabled when you select 'All non-synonymous variants' as the types of variants to be used for the burden calculations above.</token>
     </macros>
-    <expand macro="requirements" />
+    <expand macro="requirements">
+        <requirement type="package" version="0.24.2">pandas</requirement>
+    </expand>
     <expand macro="stdio" />
     <expand macro="version_command" />
     <command>
```
works fine, great detective work @wm75!
I can try bumping the old build and we can see how much of a hassle that would be.

**Comment by wm75** [comment]
Exactly the diff I just made locally :smile: Just had no time to test it yet.
Just let me know if that's the preferred solution and I'll push it.

**Comment by mvdbeek** [comment]
Fingers crossed for https://github.com/bioconda/bioconda-recipes/pull/19953. If we can get a new build that'd be better for other people possibly using this container and deployers not needing both images.

**Comment by bgruening** [comment]
We need to bump the version number, right?

**Comment by wm75** [comment]
Of course, thanks! Glad if that's all that's left to do here :smile:

**Pull request merged**